internal/pkg/cli/env_init.go

```diff
@@ -461,22 +461,28 @@ https://aws.amazon.com/premiumsupport/knowledge-center/ecs-pull-container-api-er
 		if err != nil {
 			if err == selector.ErrSubnetsNotFound {
 				log.Errorf(`No existing public subnets were found in VPC %s. You can either:
-- Create new public subnets and then import them.
-- Use the default Copilot environment configuration.`, o.importVPC.ID)
+- %s to interrupt, create new public subnets, then rerun %s and import them.
+- %s to interrupt, then rerun %s and use the default Copilot environment configuration.
+- Proceed without public subnets, knowing that deploying a load-balanced web service in this environment will fail because Load Balancers require at least two public subnets in different Availability Zones.
+`, o.importVPC.ID, color.HighlightUserInput("ctrl-c"), color.HighlightCode("copilot env init"), color.HighlightUserInput("ctrl-c"), color.HighlightCode("copilot env init"))
+			} else {
+				return fmt.Errorf("select public subnets: %w", err)
 			}
-			return fmt.Errorf("select public subnets: %w", err)
 		}
 		o.importVPC.PublicSubnetIDs = publicSubnets
 	}
 	if o.importVPC.PrivateSubnetIDs == nil {
 		privateSubnets, err := o.selVPC.PrivateSubnets(envInitPrivateSubnetsSelectPrompt, "", o.importVPC.ID)
 		if err != nil {
 			if err == selector.ErrSubnetsNotFound {
```

Comment:
I think we should enforce always at least 2 private subnets. I don't see a scenario where folks would only have public subnets 🤔 and get rid of the large log statement.
```
select private subnets: at least 2 private subnets must be provided for high availability
```

Reply:
Can we throw a warning about best practices but let them have just one private subnet? We've already had a couple of issues opened where people have tried to do this, so that means more users have tried but not opened issues.

Reply:
I think requiring two public subnets is different bc they're most likely going to need a load balancer, so that's a hard limitation.

Reply:
I was thinking we should require at least 2 for the exact same reason (enabling internal load balancers as a feature) as ingress to backend services.  
Reading #2361 I think providing a nice err message should address this situation.

Reply:
Ah, okay. That makes sense! 👍🏼